WalletWasabi/WabiSabi/Backend/Models/WabiSabiProtocolErrorCode.cs

```diff
@@ -21,6 +21,7 @@ public enum WabiSabiProtocolErrorCode
 	ScriptNotAllowed,
 	IncorrectRequestedAmountCredentials,
 	WrongCoinjoinSignature,
+	SignatureTooLong,
```

Comment:
Nop. This enumerations are used, in many cases, to let the client to know what was wrong and the problem here is that they are not serialized by name but by position so, this is a breaking change in the protocol. We can only add at the end of the list.

Reply:
Sure, I didn't realize that.

Fixed in https://github.com/zkSNACKs/WalletWasabi/pull/8831/commits/13f290ebc74f61f94c4e44a2fcf6e84bd4dd6cf3.

This is off topic, but it would be better if the coordinator returned enum encoded as string rather than integer.